src/widgets/FormControl.js

```diff
@@ -167,34 +175,40 @@ const FormControlComponent = ({
       }
     );
 
-  const Buttons = useCallback(
+  const onSaveCompletedForm = useCallback(() => onSave(completedForm), [onSave, completedForm]);
+  const SaveButton = useCallback(
+    () => (
+      <PageButton
+        onPress={onSaveCompletedForm}
+        style={localStyles.saveButton}
+        isDisabled={!canSaveForm || isDisabled}
+        textStyle={localStyles.saveButtonTextStyle}
+        text={saveButtonText}
+      />
+    ),
+    [isDisabled, showCancelButton, canSaveForm, saveButtonText, onSaveCompletedForm]
+  );
```

Comment:
done now so more of a comment for the future and to see what you reckon: I think it's ok to maybe not memo buttons as default given `PageButton` is memoized on its own, so we're possibly doubling up on all of the prop comparisons

Reply:
Hm, yeah that's a good point. I guess the only time you'd want to do it is for parameters passed as expressions (to avoid recalculating them on each render).